CHANGES.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+# 1.1.17 (core) / 0.17.17 (libraries)
+
+### New
+
+- [dagster-airflow] Has been moved to 1.x.x to denote the stability of its api's going forward.
```

Comment:
uber-nit - would suggest still leading with "The `dagster-airflow` package has been moved" instead of making a full sentence starting with [dagster-airflow]